cmd/osv-scanner/scan/image/__snapshots__/command_test.snap

```diff
@@ -2243,9 +2247,9 @@ Scanning local image tarball "../../../../internal/image/fixtures/test-node_modu
             }
           },
           "vulnerabilities": [
+            "UBUNTU-CVE-2022-3219",
```

Comment:
this looks like is caused by upstream ordering? Do we put the one with the most amount of upstream at the top or the one with the least upstream at the top? UBUNTU-CVE-2022-3219 recently added an upstream field. But i think we wanted to put USN at the top, right? @jess-lowe